backend/canisters/community/impl/src/lib.rs

```diff
@@ -240,7 +240,7 @@ impl RuntimeState {
                     next_event_expiry = Some(expiry);
                 }
             }
-            prize_refunds.extend(result.prize_refunds);
+            prize_refunds.extend(result.final_prize_payments);
```

Comment:
This should probably be named something like `payments_to_make` (feel free to come up with something better)